data/kartiklk.json

```
@@ -0,0 +1,66 @@
+{
+    "name": "Kartik khode",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Backend Developer | Open Source and Tech Enthusiast",
+    "avatar": "https://github.com/kartiklk.png",
+    "tags": ["NodeJS", "ExpressJS", "MongoDB", "Javascript", "OpenSource"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/kartikkhode21" },
+      { "icon": "FaGithub", "url": "https://github.com/Kartiklk" },
+      { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/kartik-khode-32a7a325a/" }
```

I think you are missing a command (`,`) at the end of this line

```suggestion
      { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/kartik-khode-32a7a325a/" },
```